web-client/public/js/grnstate.js

```diff
@@ -42,5 +42,8 @@ export const grnState = {
 
     dashedLine: false,
 
-    annotateLinks: () => annotateLinks(currentNetwork)
+    annotateLinks: () => annotateLinks(currentNetwork),
+
+    normalizationMax : null,
+    resetNormalizationMax: null
```

Comment:
This works, although is there a more appropriate initial value than `null`?

Reply:
as of right now, there is none because a preset normalization value does not exist until the network is actually loaded